source/sdk/Memory.ooc

```diff
@@ -2,17 +2,8 @@ include string // Memory routines in C are actually in string.h
 version(windows) { include malloc }
 else { include alloca }
 
-// GC_MALLOC zeroes the memory, so in the non-gc version, we prefer to use calloc
-// to the expense of some performance. Don't use malloc instead - existing ooc code
-// is written assuming that gc_malloc zeroes the memory.
-gc_malloc: func (size: SizeT) -> Pointer { gc_calloc(1, size) }
```

Comment:
When I try to use println from gc_malloc, I always get a crash. There should be a way to prevent infinite recursion when using dynamic memory from within the function that gives dynamic memory to the other callers.